src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/Runners/MonitorRunner.cs

```diff
@@ -156,7 +156,7 @@ public virtual async Task StartAsync(string command, string[] args, Cancellation
 
         public virtual async Task WaitForExitAsync(CancellationToken token)
         {
-            await RunnerExitedTask.WithCancellation(token).ConfigureAwait(false);
+            await TestCommon.TaskExtensions.WithCancellation(RunnerExitedTask, token).ConfigureAwait(false);
```

Comment:
Please revert. Only call extension methods in this fashion if it's absolutely necessary to disambiguate the method.